<html>
Other then just catching exceptions you can also raise them yourself with a raise statement.
<br>
In this task we assume a user has given us some values which we stored in the list 'value'<br>
Note: we hardcoded some values for you<br>
Loop through the values in the list. <br>
place the statement print("the value is :",values[cur])<br>
If the value is a string raise a ValueError that says "this is a string!"<br>
Note: you'll notice that once the exception is raised the execution of the program stops.
<br>
To check if the value  is a string you can use  if type(values[cur]) == str
<br>
<br>
Run this a few times notice how the output is not exactly as expected
</html>